modules/oe_theme_content_project/src/Plugin/ExtraField/Display/PercentageExtraField.php

```diff
@@ -106,26 +106,31 @@ public function viewElements(ContentEntityInterface $entity) {
   }
 
   /**
-   * Gets the percentage of total, without decimals.
+   * Gets the percentage of total.
    *
-   * If input values are not greater that 0, returns 0.
+   * If input values are not greater than 0, returns 0.
    *
    * @param float $total
    *   The total value.
    * @param float $part
-   *   The percentage value.
+   *   The contribution part of the total value.
    *
-   * @return float
+   * @return string
    *   Percentage value.
    */
-  protected function getPercentage(float $total, float $part): float {
+  protected function getPercentage(float $total, float $part): string {
     $percentage = 0;
 
     if ($total > 0 && $part > 0) {
-      $percentage = round(100 * $part / $total, 0);
+      $percentage = 100 * $part / $total;
     }
 
-    return $percentage;
+    // If our number is not round we will limit the decimals to one.
+    if (intval($percentage) != $percentage) {
+      return number_format($percentage, 1, '.', ' ');
+    }
+
+    return (string) $percentage;
```

Comment:
By doing this:
```suggestion
    return round($percentage, 2);
```
We would get a more accurate percentage:
<img width="613" alt="2 decimals" src="https://user-images.githubusercontent.com/22908988/170009228-278f7cd5-ab5f-4b01-bb22-5d945978ac88.png">

But we could also use only one decimal and get the same result we currently get with the if statement above that could be replaced by this:
```suggestion
    return round($percentage, 1);
```
<img width="585" alt="1 decimal" src="https://user-images.githubusercontent.com/22908988/170009537-a02246b6-327b-403f-b003-2e830b128682.png">

Reply:
round can't be applied in all cases, see my previous comment. That's why that logic was implemented.

Reply:
It will apply the decimal even if the final number is round, like 10% becomes 10.0% and this is not what we want.

Reply:
Did you check if it's the case? Because it doesn't seems so:
<img width="574" alt="1 decimal" src="https://user-images.githubusercontent.com/22908988/170209140-4d129fe3-43f9-48f6-a4d6-899129751e64.png">
<img width="546" alt="no decimal" src="https://user-images.githubusercontent.com/22908988/170209145-8b12d485-f8de-4af6-95d3-579e827f1501.png">